src/main/java/Main.java

```diff
@@ -0,0 +1,54 @@
+import db.Storage;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.Activity;
+import model.ActivityType;
+import service.ActivityReaderService;
+import service.CsvWriterService;
+import service.ReportCreator;
+import service.impl.ActivityReaderServiceImpl;
+import service.impl.CsvWriterServiceImpl;
+import service.impl.ReportCreatorImpl;
+import strategy.ActivityTypeHandler;
+import strategy.ActivityTypeStrategy;
+import strategy.impl.ActivityTypeStrategyImpl;
+import strategy.impl.AdditionTypeHandler;
+import strategy.impl.PurchaseTypeHandler;
+
+public class Main {
+    private static final String FROM_FILE_NAME = "src/files/fruit.csv";
+    private static final String TO_FILE_NAME = "src/files/report.csv";
+
+    public static void main(String[] args) {
+        ActivityReaderService activityReaderService = new ActivityReaderServiceImpl();
+        final List<Activity> activityList = activityReaderService.readFromFile(FROM_FILE_NAME);
+
+        Map<String, Integer> fruitBox = new HashMap<>();
+        fruitBox.put("apple", 0);
+        fruitBox.put("banana", 0);
```

Comment:
Solution should work with any fruits, we do not know content of file we will get as input
```suggestion
```